bindings/python/pyproject.toml

```
@@ -34,6 +34,7 @@ name = "pyiceberg-core"
 readme = "project-description.md"
 requires-python = "~=3.9"
 dynamic = ["version"]
```

```suggestion
dynamic = ["version", "license"]
```

i think we can pull it in dynamically

---

ah great catch! Looks like we currently do not have the LICENSE file in either whl or sdist. I double checked with https://pypi.org/project/pyiceberg-core/0.6.0/#files

Note the apache 2.0 license exists in both whl and sdist, but not the actual LICENSE file.